lib/libsys/Symbol.sys.map

```diff
@@ -383,7 +383,8 @@ FBSD_1.7 {
 };
 
 FBSD_1.8 {
-	 kcmp;
```

Comment:
I was going to upstream this whitespace cleanup but turns out that file does not exist in FreeBSD.